src/foam/u2/view/TableCellFormatter.js

```diff
@@ -212,8 +215,11 @@ foam.CLASS({
       class: 'foam.u2.view.TableCellFormatter',
       name: 'tableCellFormatter',
       value: function(date) {
-        // allow the browser to deal with this since we are technically using the user's preference
-        if ( date ) this.add(date.toLocaleString());
```

Comment:
I don't think we want to change this. Let the user's browser do this work for us.